src/styles/blocks/card.scss

```diff
@@ -0,0 +1,109 @@
+.card {
```

Comment:
add padding to a card for all sides to avoid left/right margin/padding for inner elements